control-plane/subcommand/sync-catalog/command.go

```diff
@@ -227,31 +255,17 @@ func (c *Command) Run(args []string) int {
 	// Start the K8S-to-Consul syncer
 	var toConsulCh chan struct{}
 	if c.flagToConsul {
-		// If namespaces are enabled we need to use a new Consul API endpoint
-		// to list node services. This endpoint is only available in Consul
-		// 1.7+. To preserve backwards compatibility, when namespaces are not
-		// enabled we use a client that queries the older API endpoint.
```

Comment:
We no longer need to maintain compatibility with that consul version so I refactored it to not use these interfaces and instead query node services directly.